### ISSUE TITLE
(feature request) specify [text] window size

### ISSUE THREAD

**Issue by mariobuoninfante** [pure-data/pure-data#801]
Hi,

When I send [click( to a [text] object this always has the same size (600x340 - from what I can see in x_text.c row 98) regarding its content.
I was wondering if we could specify it's size with something like:

[click xpos ypos(

Cheers,
Mario

**Comment by Spacechild1** [comment]
Good idea, but I would rather add a dedicated method for this. After all, you might want to resize while the window is already open. Additionally, it would be nice if we could also set the window position.
